### ISSUE TITLE
Sass - Nested items at the end of the file doesn't register final new line

### ISSUE THREAD

**Issue by bgriffith** [sasstools/sass-lint#627]
The following code will ignore the final new line and incorrectly flag up that a new line is required. This appears only to happen when the file ends with a nested node.

```sass
.foo
  content: 'foo'
  .bar
    content: 'bar'

```
Sass 1.5.1, develop branch
